README.md

```diff
@@ -174,6 +174,12 @@ Info: Chemex method
       <td align="center">String</td>
       <td align="center"><code>Custom brewing method</code></td>
     </tr>
+    <tr>
+      <td align="center"><code>--digits</code></td>
+      <td align="center">Number of digits up to which the result is to be rounded</td>
```

Comment:
`is rounded` good enough.

Reply:
fixed in https://github.com/sepandhaghighi/mycoffee/pull/25/commits/bec88b6475a6ad0503bbfbe61c1492eab60a8192